src/library/Uno.Material/Styles/Controls/v2/MediaPlayerElement.xaml

```diff
@@ -0,0 +1,673 @@
+	<x:String x:Key="RewindPathData">m11.41,16.01c0,.8-.87,1.27-1.5.81l-5.5-4.01c-.54-.39-.54-1.22,0-1.61l5.5-4.01c.63-.46,1.5,0,1.5.81v8.02Zm1.58-3.21l5.5,4.01c.63.46,1.5,0,1.5-.81V7.99c0-.8-.87-1.27-1.5-.81l-5.5,4.01c-.54.39-.54,1.22,0,1.61Z</x:String>
+	<x:String x:Key="SkipBackwardPathData">m11.97,5.51v-2.51c0-.45-.54-.67-.86-.35l-3.81,3.51c-.2.2-.2.51,0,.71l3.81,3.53c.32.31.86.09.86-.35v-2.53c3.32,0,6.02,2.69,6.02,6s-2.7,6-6.02,6c-2.98,0-5.46-2.17-5.94-5-.09-.54-.53-1-1.09-1s-1.01.45-.94,1c.49,3.95,3.87,7,7.97,7,4.44,0,8.03-3.58,8.03-8s-3.59-8-8.03-8Zm3.15,8.6c0,.32-.03.6-.1.82-.07.22-.17.42-.29.57-.12.15-.28.26-.45.33-.17.07-.37.1-.59.1s-.41-.03-.59-.1c-.18-.07-.33-.18-.46-.33-.13-.15-.23-.34-.3-.57-.07-.23-.11-.5-.11-.82v-.74c0-.32.03-.6.1-.82.07-.22.17-.42.29-.57.12-.15.28-.26.45-.33.17-.07.37-.1.59-.1s.41.03.59.1c.18.07.33.18.46.33.13.15.23.34.3.57.07.23.11.5.11.82v.74Zm-.88-1.3s0-.03,0-.04c-.03-.13-.07-.23-.12-.31s-.11-.14-.19-.17c-.06-.02-.11-.04-.17-.05-.02,0-.05,0-.08,0-.06,0-.11,0-.17.02-.01,0-.03,0-.04.01-.01,0-.03.01-.04.02-.07.03-.14.09-.19.17-.05.08-.09.18-.12.31-.03.13-.04.29-.04.48v.97c0,.19,0,.35.04.48.03.13.07.24.12.32.05.08.11.14.19.17.08.03.16.05.25.05.04,0,.08,0,.12-.01.05,0,.09-.02.13-.04.07-.03.14-.09.19-.17s.09-.19.11-.32c.02-.13.04-.29.04-.48v-.97h0c0-.15,0-.28-.02-.39,0-.02,0-.04,0-.05Zm-3.4,3.06h-.85v-3.26l-1.01.31v-.69l1.77-.63h.09v4.27Z</x:String>
+	<x:String x:Key="SkipForwardPathData">m12.03,3v2.51c-4.44,0-8.03,3.58-8.03,8s3.59,8,8.03,8c4.1,0,7.47-3.05,7.97-7,.07-.55-.39-1-.94-1s-.99.45-1.09,1c-.48,2.84-2.96,5-5.94,5-3.32,0-6.02-2.69-6.02-6s2.7-6,6.02-6v2.53c0,.45.54.67.86.35l3.81-3.53c.2-.2.2-.51,0-.71l-3.81-3.51c-.32-.31-.86-.09-.86.35Zm1.74,8.5h.02s.02,0,.02,0c.21,0,1.43-.02,1.42,1.82v.74c0,1.84-1.21,1.82-1.42,1.82h-.02s-.02,0-.02,0c-.19,0-1.42.02-1.42-1.82v-.74c0-1.84,1.22-1.82,1.42-1.82Zm.03,3.7c.38,0,.59-.26.59-1.03v-.97c0-.75-.22-1.01-.6-1.01s-.59.26-.59,1.01v.97c0,.77.22,1.03.6,1.03Zm-4.42-.54c0,.37.33.54.62.54.44,0,.65-.27.65-.59,0-.22-.05-.64-.7-.64h-.45v-.66h.46c.22,0,.63-.07.63-.59h0c0-.05,0-.54-.58-.54-.26,0-.57.14-.57.48h-.85c0-.81.78-1.16,1.4-1.16,1.39,0,1.44,1.01,1.44,1.22,0,.6-.54.87-.65.92.26.1.71.34.71,1,0,.79-.64,1.25-1.51,1.25-.55,0-1.45-.26-1.45-1.23h.85Z</x:String>
+	<x:String x:Key="VolumePathData">m4,9.17c-.55,0-1,.42-1,.94v3.78c0,.52.45.94,1,.94h2.59c.27,0,.52.1.71.28l3,2.83c.63.6,1.71.17,1.71-.67V6.72c0-.84-1.08-1.26-1.71-.67l-3,2.83c-.19.18-.44.28-.71.28h-2.59Zm12.5,2.83c0-1.12-.46-2.14-1.21-2.9-.51-.51-1.29-.08-1.29.62v4.56c0,.7.78,1.13,1.29.62.75-.76,1.21-1.77,1.21-2.9Zm-1.3-7.95c-.6-.22-1.2.24-1.2.84v.06c0,.42.29.78.7.94,2.52,1,4.3,3.36,4.3,6.1s-1.77,5.1-4.3,6.1c-.41.16-.7.52-.7.94v.06c0,.61.6,1.06,1.2.84,3.39-1.22,5.8-4.32,5.8-7.95s-2.41-6.73-5.8-7.95Z</x:String>
+
+	<Style TargetType="MediaTransportControls"
```

Comment:
We should follow the pattern that we are using for the rest of the controls when defining the default styles. Like [here in PasswordBox](https://github.com/unoplatform/Uno.Themes/blob/ab039e406f5725a88e6502fb80b773f5e3d11d91/src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml#L579-L581).

The DefaultMediaTransportControlsStyle should be defined at the bottom of the style XAML file.

And then we should add the implicit style based on DefaultMediaTransportControlsStyle in the [_Resources.xaml](https://github.com/unoplatform/Uno.Themes/blob/ab039e406f5725a88e6502fb80b773f5e3d11d91/src/library/Uno.Material/Styles/Controls/v2/_Resources.xaml#L31)